scrypt/src/params.rs

```diff
@@ -102,11 +102,20 @@ impl Params {
 
     /// log₂ of the Scrypt parameter `N`, the work factor.
     ///
-    /// Memory and CPU usage scale linearly with `N`.
+    /// Memory and CPU usage scale linearly with `N`. If you need `N`, use
+    /// [`Params::n`] instead.
     pub const fn log_n(&self) -> u8 {
         self.log_n
     }
 
+    /// `N` parameter: the work factor.
+    ///
+    /// This method returns the square of [`Params::log_n`]. Memory and CPU
```

Comment:
In my understanding, "the square of `log_n`" is `log_n^2`. I think it's better to write "2 to the power of `log_n`" or just `2^log_n`.